src/sources/util/fake.rs

```diff
@@ -70,14 +75,26 @@ pub fn apache_error_log_line() -> String {
     )
 }
 
+pub fn syslog_3164_log_line() -> String {
+    format!(
+        "<{}>{} {} {}[{}]: {}",
+        priority(),
+        timestamp_syslog_3164(),
+        username(),
```

Comment:
According to RFC 3164, this is a hostname, not a username. This will still produce a valid format, though, so I don't know if it matters.

Reply:
Ah, good catch. The Flog output has a username but that should not, of course, be the reference point here.